web/src/compose_recipient.ts

```diff
@@ -110,16 +124,18 @@ export function get_posting_policy_error_message() {
         return "";
     }
 
-    const stream = sub_store.get(compose_state.selected_recipient_id);
-    if (stream && !stream_data.can_post_messages_in_stream(stream)) {
-        return $t({
-            defaultMessage: "You do not have permission to post in this stream.",
-        });
+    if (isNumber(compose_state.selected_recipient_id)) {
```

Comment:
This should be an assert, since we already checked the `compose_state.selected_recipient_id === "direct"` case, above.

Reply:
I guess `""` is a possible value too.